test/FunctionalTests/Framework/TestExecutionLayer/AstoriaDatabase.cs

```diff
@@ -458,27 +374,71 @@ private void AttachDatabase()
 
             DropDatabase();
 
-            AstoriaTestLog.WriteLineIgnore("-------AttachDb: {0}---------", this.MachineConnectionString);
+            AstoriaTestLog.WriteLineIgnore("-------CreatePopulatedDb: {0}---------", this.MachineConnectionString);
             using (SqlConnection sqlConn = new SqlConnection(this.MachineConnectionString))
             {
                 sqlConn.Open();
                 using (SqlCommand sqlCmd = sqlConn.CreateCommand())
                 {
                     sqlCmd.CommandTimeout = 0;
+                    sqlCmd.CommandText = String.Format(
+                        "CREATE DATABASE [{0}]\n" +
+                        " CONTAINMENT = NONE\n" +
+                        " ON PRIMARY\n" +
+                        "(NAME = N'{0}', FILENAME = N'{1}', SIZE = 4288KB, MAXSIZE = UNLIMITED, FILEGROWTH = 1024KB)\n" +
+                        " LOG ON\n" +
+                        "(NAME = N'{0}_log', FILENAME = N'{2}', SIZE = 3456KB, MAXSIZE = 2048GB, FILEGROWTH = 10 %)",
+                        this.DatabaseName,
+                        this.AttachedMdf,
+                        this.AttachedLdf);
+                    sqlCmd.ExecuteNonQuery();
+                }
 
-                    CopyDatabaseFiles();
+                String filePath = FindDatabaseFile(this.DatabasePrefixName + ".sql");
+                using (StreamReader reader = new StreamReader(filePath))
+                using (SqlCommand sqlCmd = sqlConn.CreateCommand())
+                {
+                    // Read from the file until it is empty.
+                    StringBuilder sqlStatements = new StringBuilder();
+                    String fileLine = reader.ReadLine();
+                    while (fileLine != null)
+                    {
+                        // If we find a "GO", execute everything we received up
+                        // to this point and ignore the "GO".
+                        if (fileLine == "GO")

```

Comment:
The files comes from SSMS's scripting tool and includes GO as the last line. I agree that is a possibility but I'd rather change the SQL if needed then make this code handle that case.